.dockerignore

```diff
@@ -29,3 +29,4 @@ vue/babel.config*
 vue/package.json
 vue/tsconfig.json
 vue/src/utils/openapi
+venv/
```

Comment:
Just a small fix for the docker build. In the `.gitignore` the venv is already excluded, but it also has to be done for docker, otherwise the build will fail, when a venv is used.

Reply:
not sure, it has not been ignored for a long time and everything worked, I need to look into that but we should seperate the change from changing the importer error message as these two things have nothing to do with each other.

Reply:
Yes, you are right. I removed this and will create a separate issue to track this.